e2e-tests/haproxy/run

```diff
@@ -53,10 +67,11 @@ main() {
 	check_haproxy_writer
 
 	desc 'delete active writer and checking all haproxy pods still point to the same writer'
-	kubectl_bin delete pod $cluster-pxc-0
-	sleep 3
+	desc 'fail pod-0 pod for 60s'
+	fail_pod $cluster-pxc-0
```

Comment:
could we add pxc-pod-0 in desc? Just to be clear.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/commit/b75e2adadaa2de5956475a511a7f19d264ae8785 done